app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,11 @@
 <resources>
-  <string name="app_name">Backpack Android</string>
   <string name="main_button">Show me more!</string>
-
+  <string name="app_name">Backpack</string>
+  <string name="stub" translatable="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```

Comment:
while we're at it some proper copy can be added here, like Backpack is awesome etc

Reply:
@shaundon can I borrow one of your poems here?